packages/core-web/src/styles/markbind.css

```diff
@@ -59,10 +58,25 @@ pre > code.hljs[heading] {
     display: block;
 }
 
-code > span.highlighted {
+code > span.highlighted-light {
```

Comment:
actually, since you already havemutually exclusive stylesheets, how about moving all code related styles into `github.min.css` / `vs2015.css` (renaming appropriately and preserving license notices)?

should be easier to implement than requiring separate classes for light / dark

Reply:
Good idea, I'll try to move the theme-specific styling to their respective CSS files, but some will be kept in `markbind.css` for identical things like layouting and common colors (if any) so that it can be shared by both.

I'll also keep the license but add something like `(slightly modified for MarkBind)` to preserve credits